### PR TITLE
weaviate-cli 3.0.2 (new formula)

### PR DESCRIPTION
**Pull request by Dabz**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by Dabz** [comment]
@chenrui333 , this PR should be good for review. Sorry for the CI/CD strugle, Linux give me a hard time ;)

File: Formula/w/weaviate-cli.rb [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+  url "https://files.pythonhosted.org/packages/e2/aa/d49ae3e5d399d3bb4026188017220c4dcdd240d64de88df41259056c801a/weaviate_cli-3.0.2.tar.gz"
+  sha256 "7a4a3ced248678a6d006ce849edf1d97b15237605f2735bb5ea7882b954c1fdb"
+  license "BSD-3-Clause"
+
+  depends_on "ninja" => :build
+  depends_on "pkgconf" => :build
+  depends_on "rust" => :build
+  depends_on "cryptography"
+  depends_on "openssl@3"
+  depends_on "python@3.13"
```
Comment (chenrui333) [inline Formula/w/weaviate-cli.rb:15]: you also need this

```suggestion
  depends_on "python@3.13"

  uses_from_macos "libffi"
```

Comment (Dabz) [reply Formula/w/weaviate-cli.rb:15]: Good to know, I added the `uses_from_macos "libffi"`

**Review by daeho-ro — APPROVED** [review]
Looks good to me.

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@Dabz thanks a lot for your contribution!!

**Pull request merged**